### PR TITLE
Feature testing black

### PR DESCRIPTION
**Pull request by regislon**
# Prework

- [x] I understand and agree to this repository's [code of conduct](https://github.com/ltelab/disdrodb/blob/main/CODE_OF_CONDUCT.md).
- [x] I understand and agree to this repository's [contributing guidelines](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.md).
- [x] I have already submitted an [issue](https://github.com/ltelab/disdrodb/issues) or [discussion thread](https://github.com/ltelab/disdrodb/discussions) to discuss my idea with the maintainers.

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [x] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [x] Yes
- [ ] No

If yes, please describe the impact and communicate accordingly:

**The PR fulfills these requirements:**

- [x] It's submitted to the branch named as follow : 
  - Add a reader: `reader-<institute>-<campaign>`
  - Fix a bug: `bugfix-<some_key>-<word>`
  - Improve the doc: `doc-<some_key>-<word>`
  - Add a new feature: `feature-<some_key>-<word>`
  - Refactor some code: `refactor-<some_key>-<word>`
  - Optimize some code: `optimize-<some_key>-<word>`
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.
- [x] All tests are passing.
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

black 

**Other information:**

# Related GitHub issues and pull requests

- Ref: #66 and #23

### CODE REVIEW

**Comment by regislon** [comment]
Issue ! Black fails even though I run it on my local (and committed) files. I'm investigating that

**Comment by regislon** [comment]
Hi @charlottegiseleweil could you please have a look at this PR ? I run black over the entire folder with the latest version of black in both env (local + github). As you can see, It succeed now. Since lots of files are impacted in the PR, could you please valide it quickly ?

File: disdrodb/test_balck.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import os
```
Comment (ghiggi) [inline disdrodb/test_balck.py:1]: This file must be removed

File: disdrodb/L0/templates/FRANCE/temp_parser_ACTRIS-CCRES.py [hunk 2]
```diff
@@ -387,6 +387,7 @@
```
Comment (ghiggi) [inline disdrodb/L0/templates/FRANCE/temp_parser_ACTRIS-CCRES.py:387]: This file must be removed ... also the folder templates

**Review by ghiggi — COMMENTED** [review]
LGTM @regislon

File: disdrodb/L0/templates/FRANCE/temp_parser_ACTRIS-CCRES.py (hunk 2 above)
Comment (regislon) [reply disdrodb/L0/templates/FRANCE/temp_parser_ACTRIS-CCRES.py]: sorry, it is done yet

**Comment by charlottegiseleweil** [comment]
@regislon lmk if you still need my input (sorry long day!)

**Pull request merged**